salt/orch/etcd-migrate.sls

```diff
@@ -0,0 +1,84 @@
+# Generic Updates
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
+
+update_grains:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_grains
+
+update_mine:
+  salt.function:
+    - tgt: '*'
+    - name: mine.update
+    - require:
+       - salt: update_pillar
+       - salt: update_grains
+
+master-stop-services:
+  salt.state:
+    - tgt: 'roles:kube-master'
+    - tgt_type: grain
+    - sls:
+      - kube-apiserver.stop
+      - kube-controller-manager.stop
+      - kube-scheduler.stop
+      - etcd.stop
+    - require:
+       - salt: update_mine
+
+worker-stop-services:
+  salt.state:
+    - tgt: 'roles:kube-minion'
+    - tgt_type: grain
+    - sls:
+      - kubelet.stop
+      - kube-proxy.stop
```

Comment:
Nit: I still don't think we need to or should be stopping kube-proxy, but will circle back later :)